src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        printGreeting();
+
+        try {
+            loadTasks();
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            String line = in.nextLine();
+            String taskType = "";
+
+            try {
+                if (line.equals("list")) {
+                    printList();
+                } else if (line.equals("bye")) {
+                    printBye();
+                    break;
+                } else if (line.contains("done")) {
```

Comment:
This line of code is a bit dangerous. You set the type of the task to be "done" if there is a done in the command. What if I type in a todo command "todo done" or anything that contains two or more keywords? In this script, the "todo done" command will be seen as a done command, as you used the "else if" statements to check.